reqs/management/commands/import_reqs.py

```diff
@@ -219,12 +219,32 @@ def __init__(self, keywords=None):
         self.keywords = KeywordProcessor(keywords)
         self.connections = []
         self.req_ids = set()
+        self.last_id = "0.0"
+
+    def fix_excel_decimals(self, reqid):
+        """
+        Excel strips trailing zeroes, so for example what should be 1.20 is 1.2
+        in the data.
+        Since the rows are sequential (or should be!), we should be able to
+        account for this.
+        """
+        policy, requirement = reqid.split(".")
+        last_policy, last_requirement = self.last_id.split(".")
+        if policy == last_policy:
+            if last_requirement.endswith("9"):
```

Comment:
Minor: this code would be less pointy if we used `if policy... and last_req...:`